### PR TITLE
Mention of new Symfony client

### PR DESCRIPTION
**Pull request by mathiasstocker**
There was no mention of the new `symfony` client which clearly should be mentioned under "Getting Started".

### CODE REVIEW

File: setup.rst [hunk 1]
```diff
@@ -16,6 +16,7 @@ Installing Symfony
 Before creating your first Symfony application, make sure to meet the following
 requirements:
 
+* Installed :doc:`Symfony client <https://symfony.com/download>`
```
Comment (noniagriconomie) [inline setup.rst]: Maybe write, optional but recommended?

this tool is not a requirement, just an new option (even if cool/easy etc)

**Comment by javiereguiluz** [comment]
I'm afraid there's been some confusion. The first item you want to add to the list ... is already in that list ... as the last item:

image

**Comment by mathiasstocker** [comment]
You are right!

**Comment by javiereguiluz** [comment]
Note ... although this is closed, your comment made me realize something about this which is confusing. But this can't be solved in the Symfony Docs alone. I've created a discussion in another internal Symfony repository to try to simplify things.

**Pull request closed without merging**